### ISSUE TITLE
New UI: Cannot Add Tutorial

### ISSUE THREAD

**Issue by 96LawDawg** [ArnoldSmith86/virtualtabletop#1487]
I'm trying to update the Arrays tutorial after moving to v9 and the new `numericSort`.  I can add the Tutorial, but it only goes into Games and not into Tutorials.  The mode is tutorial, but that doesn't make it work.  I don't know how to make this work.  Here is the file if you want to test it out.

[Tutorial - Arrays.zip](https://github.com/ArnoldSmith86/virtualtabletop/files/10329140/Tutorial.-.Arrays.zip)

**Comment by ArnoldSmith86** [comment]
Yeah, not everything will work from the UI. But you can simply move the folder from library/games to library/tutorials.

**Comment by ArnoldSmith86** [comment]
Also note that the tutorials can now be part of the feature PR itself.
